common/src/main/java/org/apache/comet/parquet/ColumnReader.java

```diff
@@ -126,6 +126,13 @@ public void setPageReader(PageReader pageReader) throws IOException {
     }
   }
 
+  /** This method is called from Apache Iceberg. */
+  public void setRowGroupReader(RowGroupReader rowGroupReader, ParquetColumnSpec columnSpec)
```

Comment:
It would be good to have some unit tests in Comet for the methods intended to be called from Iceberg, so that we catch any regressions in behavior. This could be added as a separate PR so that we don't slow down progress on the integration work.

Reply:
I filed https://github.com/apache/datafusion-comet/issues/1928